.github/workflows/test-runner-on-perf.yml

```diff
@@ -109,10 +110,23 @@ jobs:
           echo "Apply helm updates, including updated data-repo version"
           helmfile apply
           cd ${GITHUB_WORKSPACE}/${workingDir}
-      - name: "Cycle pods to clear lingering artifacts if not doing version bump"
-        if: ${{ steps.read_property.outputs.LATEST_VERSION == steps.read_property.outputs.CURRENT_SEMVER }}
+      - name: Clear Perf Database
+        env:
+          ORG_GRADLE_PROJECT_dbDatarepoUri: jdbc:postgresql://localhost:5434/datarepo
+          ORG_GRADLE_PROJECT_dbDatarepoPassword: ${{ secrets.PERF_DB_DATAREPO_PWD }}
+          ORG_GRADLE_PROJECT_dbStairwayUri: jdbc:postgresql://localhost:5434/stairway
+          ORG_GRADLE_PROJECT_dbStairwayPassword: ${{ secrets.PERF_DB_DATAREPO_PWD }}
+        run: |
+          # port foward to perf database
+          kubectl port-forward --namespace perf svc/perf-jade-gcloud-sqlproxy 5434:5432 & \
+          sleep 10 && \
```

Comment:
Why end each line with `&&` here? I think the only line that can fail is `gradlew dropAll` and it's the last command in the block.